re/sink/sinks/mainflux.json

```diff
@@ -0,0 +1,17 @@
+{
+  "id": "demo",
+  "sql": "select * from demo where v > 1.2;",
```

Comment:
What's this confing/example file? I see some hard-coded stuff, this is why I am asking.